src/Pages/Overview/App.js

```diff
@@ -0,0 +1,217 @@
+import React, { Component } from 'react'
+import './App.css'
+import InfoCard from '../Profile/admin/AdminView'
+import { Modal, ModalHeader, ModalBody, ModalFooter, Button } from 'reactstrap'
+import axios from 'axios'
+const display = require('./DisplayProfile.js')
+
+class Login extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      // All users array, update by callDatabase
+      users: [],
+      search: ''
+    }
+  }
+
+  componentDidMount () {
+    const token = window.localStorage
+      ? window.localStorage.getItem('jwtToken')
+      : ''
+
+    // Immediately direct to /login if no jwtToken token present
+    if (!token) {
+      if (this.props.history) this.props.history.push('/login')
+      return
+    }
+
+    // verify that user is login
+    axios
+      .post('/api/user/verify', { token })
+      .then(res => {
+        this.setState(
+          {
+            currentUser: res.data.email,
+            currentUserLevel: res.data.accessLevel,
+            authToken: token
+          },
+          () => {
+            this.callDatabase()
+          }
+        )
+      })
+      .catch(() => {
+        if (this.props.history) this.props.history.push('/login')
+      })
+  }
+
+  callDatabase () {
+    axios
+      // get all user!
+      .post('/api/user/search', {
+        // don't need email
+        token: this.state.authToken
+      })
+      .then(result => {
+        if (result.status >= 200 && result.status < 300) {
+          this.setState({ users: result.data })
+        }
+      })
+      .catch(err => {
+        console.log(err)
+      })
+  }
+
+  updateSearch (event) {
+    this.setState({ search: event.target.value })
+  }
+
+  /*
+  Delete api
+  parameter: Json object of object to be deleted
+  */
+  deleteUser (user) {
+    // if they decide to delete themselves: delete->logout
+    if (user.email === this.state.currentUser) {
```

Comment:
the axios call in the if and else statement is exaclty the same. instead, move the call ouside of the if/else block, and just keep the logout stuff in a single if statement